build-system/build.conf.js

```diff
@@ -26,9 +29,59 @@ const defaultPlugins = [
   require.resolve('./babel-plugins/babel-plugin-transform-version-call'),
 ];
 
+/**
+ * @return {Array<string|Object>} the minify-replace plugin options that can be
+ * pushed into the babel plugins array
+ */
+function getReplacePlugin() {
+  /**
+   * @param {string} defineStr the define flag to parse
+   * @return {Object} replacement options used by minify-replace plugin
+   */
+  function createReplacement(defineStr) {
+    const strSplit = defineStr.split('=');
+    const identifierName = strSplit[0];
+    // if no value is defined, set to true
+    const value = strSplit.length > 1 ? strSplit[1] === 'true' : true;
+
+    return {
+      identifierName,
+      replacement: {
+        type: 'booleanLiteral',
+        value,
+      },
+    };
+  }
+
+  const replacements = [];
+  // default each experiment flag constant to false
+  Object.keys(experimentsConfig).forEach(experiment => {
+    const experimentDefine = experimentsConfig[experiment]['define'];
+    if (experimentDefine) {
+      replacements.push(createReplacement(experimentDefine + '=false'));
+    }
+  });
+
+  // override define values from passed in flags
+  if (Array.isArray(argv.define)) {
```

Comment:
Does this mean that we support compiling multiple experiments at the same time? I don't find that feature to be very useful. A bit confusing instead. Can we only support one value at each build? Thanks.

Reply:
Done, will now throw an error if more than one `define` flag is passed in